conf/services/ated.yml

```diff
@@ -1,13 +1,42 @@
 serviceName: Annual Tax on Enveloped Dwellings
-serviceHeaderName: Submit and view your ATED returns
 serviceDescription: |
   This service allows users to submit and view their returns for Annual Tax on Enveloped Dwellings (ATED).
   ATED is an annual tax payable mainly by companies that own UK residential properties valued at more than £500,000 each.
   These are called enveloped dwellings.
 serviceDomain: www.tax.service.gov.uk
 serviceUrl: /ated/home
 contactFrontendServiceId: ATED
-complianceStatus: noncompliant
+complianceStatus: partial
+accessibilityProblems:
+  - the required format for the input is unclear.
+  - fieldset is missing a legend.
+  - links were not descriptive.
+  - focus order becomes illogical when the sercice timeout is appears.
```

Comment:
Spelling mistake of 'service' ?

Reply:
good spot, fixed now 👍

Reply:
I have just pushed a commit to correct this, but it's not showing

Reply:
Still needs attention as it does not make sense : focus order becomes illogical when the service timeout is appears.

Reply:
Reviewed and checked locally - updated